packages/e2e/test/providers/StakePoolProvider.test.ts

```diff
@@ -420,42 +422,45 @@ describe('StakePoolProvider', () => {
         });
       });
 
-      describe('sort by apy', () => {
+      describe.each(['lastRos', 'ros'] as const)('sort by %s', (field) => {
```

Comment:
This should probably be part of the commit that updates the provider in cardano-services

Reply:
Squashed in b4b1ca188